### ISSUE TITLE
Bug in "History's Yesterday" and add more "History" tree elements

### ISSUE THREAD

**Issue by JokoeSukri** [fmd-project-team/FMD#78]
I feel there is a limit with History on FMD, until "Last Month" limit. I using FMD nearly 2 years, and reaching nearly manga 6000 titles, sometimes my memory is blur to recalling my old manga title and still remembering the date, So I need new history feature, but instead I found something regarding Bug, and adding a proposal to enhance "History"

### **- Bug :**
![Screenshot_28](https://user-images.githubusercontent.com/32964994/55842968-b739e400-5b70-11e9-997e-a3cc0f6028f0.jpg)
Yesterday Panel is Empty

### **- Enhance :**
- [add] Yearly (ex : 2016, 2017, 2018, 2019) {with expand and collapse button}
- [add] Monthly (ex : January - December) {under sub-tab Yearly}
- [add] Weekly (ex : week I - Week IV) {under sub-tab monthly}
- [add] Auto generate, "Yearly, Monthly, Weekly" when the date is reaching end (ex : when the time reach "01 Jan 2020 - 00:00" ; Auto generate new folder "2020";"January";"Week I")
**example**
![Screenshot_26](https://user-images.githubusercontent.com/32964994/55843023-0253f700-5b71-11e9-8a37-f7fbd3d04f18.jpg)
_*nearly accurate example, History Browser use This_

**End Note : If this Proposal Enhance too much hassle, and brain wrecked. You can reject this proposal.**

**Comment by SDXC** [comment]
I fixed the "Yesterday" filter and added some more:

![DateFilters](https://user-images.githubusercontent.com/5290260/56077688-5e23b780-5ddf-11e9-8c28-49eb7211a9d2.png)

I won't implement the auto generating filters and folders. I still hope with the additional filters you're at least a bit happier than before.

**Comment by SDXC** [comment]
Version 0.9.163.0 has been released.

**Comment by JokoeSukri** [comment]
@SDXC , I like the current version, But I think you should added range view between history, not sum it all
- Today = Range Between 0 Sec to 24 hours
- Yesterday = Range Between 25 Hours to 48 Hours
- One Week = Range Between 49 Hours to 7 Days
- One Month = Range Between 8 Days to 30/31 Days
- One Year = Range Between 1 Month to 12 Months
- Year Two = 12 to 24 Months
- Year Three = 25 to 36 Months
and So on

**Comment by SDXC** [comment]
@JokoeSukri You're right, makes a bit more sense. I just more or less blindly followed the existing entries. I will change them to ranged entries.
